src/Microsoft.AspNet.OData.Shared/Builder/InstanceAnnotationDictionaryAnnotation.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.AspNet.OData.Common;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.AspNet.OData.Builder
+{
+    /// <summary>
+    /// This annotation indicates the mapping from a <see cref="IEdmStructuredType"/> to a <see cref="PropertyInfo"/>.
+    /// The <see cref="IEdmStructuredType"/> is an open type and the <see cref="PropertyInfo"/> is the specific
```

Comment:
>an open type [](start = 48, length = 12)

open type is not correct in this comment?

Reply:
#Resolved

Reply:
#Resolved